kepcoApp/Db/kepco.db

```diff
@@ -253,26 +253,6 @@ record(ao, "$(P)FIELD:SP:RBV")
     info(INTEREST, "MEDIUM")
 }
 
-record(mbbi, "$(P)CURRENT:RANGE") {
-    field(DESC, "The programmed current range")
-    field(INP, "@kepco.proto readCurrentRange($(P)) $(PORT)")
-    field(DTYP, "stream")
-    field(SCAN, "Passive")
-
-    field(ONST, "FULL")         field(ONVL, "1")
-    field(FRST, "QUARTER")        field(FRVL, "4")
-}
-
-record(mbbo, "$(P)CURRENT:RANGE:SP") {
-    field(DESC, "The current range setpoint")
-    field(OUT,  "output_link")
-    field(ONST, "FULL")         field(ONVL, "1")
-    field(FRST, "QUARTER")        field(FRVL, "4")
-    field(OUT, "@kepco.proto setCurrentRange($(P)) $(PORT)")
-    field(DTYP, "stream")
-    field(SCAN, "Passive")
-}
-
```

Comment:
did you mean to remove this? looks like we still have the calls in the proto file

Reply:
I did mean to, it's information that is no longer required for the testing we were doing and was just another device query that could cause issues